client/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "talk-lightning",
+  "homepage": "https://lionel-rowe.github.io/test-fullstack/",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.12",
+    "@fortawesome/free-solid-svg-icons": "^5.6.3",
+    "@fortawesome/react-fontawesome": "^0.1.3",
+    "@reach/router": "^1.2.1",
+    "firebase": "^5.9.4",
+    "firebaseui": "^3.5.2",
+    "node-sass": "^4.11.0",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-scripts": "2.1.8",
+    "sweetalert": "^2.1.2"
+  },
+  "devDependencies": {
+    "gh-pages": "^2.0.1",
+    "jest-dom": "^3.1.3",
+    "react-testing-library": "^6.1.2"
+  },
+  "scripts": {
+    "start": "react-scripts start",
```

Comment:
In our "README.md", there is:

> Do not use any scaffolding tool such as `create-react-app`, or any CSS framework

Reply:
I assumed this was optional as it comes under "Advanced requirements" and states "You can safely skip them if you are not asked to do any".

I could try ejecting and removing unused dependencies, but I don't think it would have much effect on bundle size, as the default `create-react-app` configuration does a pretty good job of tree shaking. Usually I only `eject` if I have a specific reason to do so.